### ISSUE TITLE
Keyboard shortcut to copy directly to clipboard

### ISSUE THREAD

**Issue by doctorsirius** [octref/polacode#70]
Cool extension, love it, but it's still faster to use the snipping tool (I know, not as beautiful), the thing is, VSCode are mostly powerusers by definition, it's cool to have a preview but it's slower, would be great if you alternatively provide a shortcut (and even a option on the dropdown menu), so after you select the code just use the keyboard shorcurt and the result is directly copied to the clipboard. It's much faster and convenient.

Thanks for your work.

**Comment by bramjoosten** [comment]
Would be cool indeed. I use it for screenshots that are too tall to fit in a single window. My use case is to send screenshots in whatsapp. Not sure about the snipping tool because I'm on mac.

**Comment by YouTwitFace** [comment]
@bramjoosten you're able to save it as a png

**Comment by angelhdz** [comment]
Yes! A button to copy to clipboard, and the corresponding **Command Palette** command.

**Comment by aNickzz** [comment]
Yeah without this it makes more sense to just use the screen clipping tool rather than the extension. This is _especially_ true if your using vscode remote through ssh or something, though I can understand if remote support is not a prority :)

**Comment by GalDayan** [comment]
Without this feature it's not useful. Windows snapshot better for capturing code scripts

**Comment by mreduar** [comment]
I found an extension that already implements this very useful feature.
https://marketplace.visualstudio.com/items?itemName=adpyke.codesnap
